libs/utils/trace.py

```diff
@@ -256,16 +256,17 @@ def __parseTrace(self, path, tasks, window, normalize_time, trace_format):
             raise ValueError('The trace does not contain useful events '
                              'nor function stats')
 
-        # Setup internal data reference to interesting events/dataframes
-
-        self._sanitize_SchedLoadAvgCpu()
-        self._sanitize_SchedLoadAvgTask()
-        self._sanitize_SchedCpuCapacity()
-        self._sanitize_SchedBoostCpu()
-        self._sanitize_SchedBoostTask()
-        self._sanitize_SchedEnergyDiff()
-        self._sanitize_SchedOverutilized()
-        self._sanitize_CpuFrequency()
+        # Santization not possible if platform missing
+        if not self.platform:
```

Comment:
I completely agree with the commit message title, but this is like using an axe instead of a bistoury... even when `platform` data are not available we can still (and have to) do many of the trace sanitation updates.
I would better move this check inside each and every `_sanitize_XXX` method to carefully cut out what is not possible.

At the same time, this update requires also to cross check all the analysis and plotting APIs to do the same.
Indeed, right now we assume that we have all the columns that are added by the availability of platform data. This should be done carefully and, since it's a big job by itself, I would rather suggest to these updates on a PR by their own.

Can you remove this patch from this series for the time being while I work on a separate PR to properly remove the platform data dependency? I've just created #427 to track this cleanups.